app/forms.py

```
@@ -74,10 +74,16 @@ class Meta:
         validators=[DataRequired(), Length(max=250)],
         render_kw={"placeholder": "Escriba o seleccione..."}
     )
-    cantidad = DecimalField('Cantidad', validators=[DataRequired()])
+    cantidad = DecimalField(
```

Flake8 (7.2.0)

[error] undefined name 'DecimalField'

(F821)